### PR TITLE
[DOC] should notice that need setting the full path when call the resource file #11620

### LINKED ISSUE #11620: [DOC] should notice that need setting the full path when call the resource file
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/dolphinscheduler/issues?q=is%3Aissue) and found no similar issues.

### What happened

serveral colleagues reported that when calling the resource file in shell task, they thought is was the file name by default, but actually the file was stored in different suddirectorie at various levels, then the failed task show cannot find the resource files.
So i suggest it should be better to notice user to using the full path.

### What you expected to happen

The doc can clearly guide users.

### How to reproduce

Just check doc.

### Anything else

_No response_

### Version

dev

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by DarkAssassinator**
## Purpose of the pull request

fix #11620 

## Verify this pull request

This pull request is code cleanup without any test coverage.

*(or)*

This pull request is already covered by existing tests, such as *(please describe tests)*.

(or)

This change added tests and can be verified as follows:

(or)

If your pull request contain incompatible change, you should also add it to `docs/docs/en/guide/upgrede/incompatible.md`

### CODE REVIEW

File: docs/docs/en/guide/project/workflow-definition.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ Drag from the toolbar <img src="../../../../img/tasks/icons/shell.png" width="15
 2. Check `Normal` for `Run Flag`. If `Prohibit Execution` is checked, the task will not execute when the workflow runs.
 3. Select `Task Priority`: when the number of worker threads is insufficient, high priority tasks will execute first in the execution queue, and tasks with the same priority will execute in the order of first in, first out.
 4. Timeout alarm (optional): check the timeout alarm, timeout failure, and fill in the "timeout period". When the task execution time exceeds **timeout period**, an alert email will send and the task timeout fails.
-5. Resources (optional). Resources are files create or upload in the `Resource Center -> File Management` page. For example, the file name is `test.sh`, and the command to call the resource in the script is `sh test.sh`.
+5. Resources (optional). Resources are files create or upload in the `Resource Center -> File Management` page. For example, the file name is `test.sh`, and the command to call the resource in the script is `sh test.sh`.Please note that you need to use the full path to the resource files.
```
Comment (EricGao888) [inline docs/docs/en/guide/project/workflow-definition.md]:
```suggestion
5. Resources (optional). Resources are files created or uploaded in the `Resource Center -> File Management` page. For example, the file name is `test.sh`, and the command to use the resource in the script is `sh test.sh`. Please note that you need to use the full path to the resource files.
```

File: docs/docs/en/guide/resource/file-manage.md [hunk 2]
```diff
@@ -65,6 +65,8 @@ In the workflow definition module of project Manage, create a new workflow using
 
 - Script: 'sh hello.sh'
 - Resource: Select 'hello.sh'
+> Notice: When invoking a resource file in the script, the file name needs to be the same as the full path of the selected resource:
```
Comment (EricGao888) [inline docs/docs/en/guide/resource/file-manage.md]:
```suggestion
> Notice: When using a resource file in the script, the file name needs to be the same as the full path of the selected resource:
```

File: docs/docs/en/guide/resource/file-manage.md [hunk 3]
```diff
@@ -65,6 +65,8 @@ In the workflow definition module of project Manage, create a new workflow using
...
+> For example: if the resource path is `/resource/hello.sh`, you need to use the full path of `/resource/hello.sh` to call in the script.
```
Comment (EricGao888) [inline docs/docs/en/guide/resource/file-manage.md]:
```suggestion
> For example: if the resource path is `/resource/hello.sh`, you need to use the full path of `/resource/hello.sh` to use it in the script.
```

File: docs/docs/zh/guide/project/workflow-definition.md [hunk 4]
```diff
@@ -16,7 +16,7 @@
   1. “运行标志”勾选“正常”，若勾选“禁止执行”，运行工作流不会执行该任务；
   1. 选择“任务优先级”：当 worker 线程数不足时，级别高的任务在执行队列中会优先执行，相同优先级的任务按照先进先出的顺序执行；
   1. 超时告警（非必选）：勾选超时告警、超时失败，填写“超时时长”，当任务执行时间超过**超时时长**，会发送告警邮件并且任务超时失败；
-  1. 资源（非必选）：资源文件是资源中心->文件管理页面创建或上传的文件，如文件名为 `test.sh`，脚本中调用资源命令为 `sh test.sh`；
+  1. 资源（非必选）：资源文件是资源中心->文件管理页面创建或上传的文件，如文件名为 `test.sh`，脚本中调用资源命令为 `sh test.sh`.注意调用需要使用资源的全路径；
```
Comment (EricGao888) [inline docs/docs/zh/guide/project/workflow-definition.md]:
```suggestion
  1. 资源（非必选）：资源文件是资源中心->文件管理页面创建或上传的文件，如文件名为 `test.sh`，脚本中调用资源命令为 `sh test.sh`。注意调用需要使用资源的全路径；
```

**Review by EricGao888 — COMMENTED** [review]
Overall LGTM, except few nits.

File: docs/docs/en/guide/project/workflow-definition.md (hunk 1 above)
Comment (DarkAssassinator) [reply docs/docs/en/guide/project/workflow-definition.md]: done

File: docs/docs/en/guide/resource/file-manage.md (hunk 2 above)
Comment (DarkAssassinator) [reply docs/docs/en/guide/resource/file-manage.md]: done

**Comment by DarkAssassinator** [comment]
done. Please review again. thx

**Review by EricGao888 — APPROVED** [review]

**Comment by EricGao888** [comment]
> done. Please review again. thx

LGTM, thanks.

**Pull request merged**